letsencrypt/client/tests/auth_handler_test.py

```diff
@@ -437,7 +439,7 @@ def _sat_incremental(self):
             # Only solve one of "0" required challs
             self.handler.responses["0"][1] = "onecomplete"
             self.handler.responses["0"][3] = None
-            self.handler.responses["1"] = ["null", "null", "goodresp"]
+            self.handler.responses["1"] = [None, None, "goodresp"]
```

Comment:
I am fairly certain these "null" to None transformations are incorrect. "null" is what is expected in the spec when a challenge is not filled in.  None is an internal check for "still needs a response".